custom_components/teslafi/model.py

```diff
@@ -241,7 +241,15 @@ def is_defrosting(self) -> bool | None:
         return (
             self.get("is_front_defroster_on") == "1"
             or self.get("is_rear_defroster_on") == "1"
-            or self.get("defrost_mode", "0") != "0"
+            or ((d := self.get("defrost_mode", "0")) != "0" and d)
```

Comment:
I think the reason this broke is that `defrost_mode` used to be `"0"`, and now it's `null`. Python's get() default value only applies if the key _is not present_ in the dict:
```
>>> d = {"defrost_mode": None}
>>> d.get("defrost_mode", "0") != "0"
True
>>> print(d.get("defrost_mode", "0"))
None
```

So (assuming that they continue returning `null` for off) this could also have been just:
```python
or self.get("defrost_mode")
```

What makes that a little weird though is because in Python, the int `0` is falsy, but string `"0"` is truthy. So if they did change it back to return "0", then that would again be seen as true so we'd be back in the same situation.

So I think what you have here now will keep working in both scenarios.